src/service/extension-location.js

```diff
@@ -49,13 +49,19 @@ export function calculateExtensionScriptUrl(location, extensionId, isLocalDev) {
  * @param {string} entryPoint
  * @param {boolean=} isLocalDev
  * @param {boolean=} opt_rtv
+ * @param {boolean=} opt_minified
  * @return {string}
  */
 export function calculateEntryPointScriptUrl(
-    location, entryPoint, isLocalDev, opt_rtv) {
+    location, entryPoint, isLocalDev, opt_rtv, opt_minified) {
```

Comment:
To be able to run `gulp test` without `--compiled`. Currently we always serve the minified version which requires `gulp dist` to be run first.

We may be able to remove this if unit tests no longer require `gulp build`.